<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutSousChap</class>
 <widget class="QDialog" name="AjoutSousChap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>221</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="txt_nom">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txt_description">
      <property name="maximumSize">
       <size>
        <width>5000</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxChap"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="titreLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SousChap Ajout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Chapitre</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
